### ISSUE TITLE
Change haptic feedback length to Android system default.

### ISSUE THREAD

**Issue by gjeuken** [MrStahlfelge/lightblocks#20]
As different (Android) devices have different quality haptic motors, they also have different optimum haptic feedback length.

For my Pixel 3a, which has a very good motor for example, the default length is 10ms and that feels much better than the 20ms the game is set to now. Subjectively a shorter feedback feels snappy and bouncy, like a key press, where a longer one will feel buzzy.

One option is to make it user adjustable, but another is to use the (Android's) system default haptic feedback length, which I believe is set by the manufacturer and so should be optimum for the haptic motor installed on the device.

**Comment by MrStahlfelge** [comment]
Thank you for your suggestion.

I can confirm your experiences: for my day to day device, 10 ms feels better, too, but on some devices the viration does not even start on 10 ms.

As far as I know, Android supports the new haptic feedback defaults since version 10. Unfortunately, I am still on Android 9 with my newest device, so I can't test the behaviour myself. Same on iOS: the feature is available, but not on the OS versions my devices run on. Anyway, I am not sure if every user is happy with the setting made by the manufacturer, though. Which brings us to a configurable length.

I thought about making the haptic feedback freely configurable already when I added the feature. I didn't implement it that way mainly because of two reasons: I was not sure if there are users even wanting to use the haptic feedback, and I was afraid it would make the settings screen even more cluttered. Looking at the analytics, 1% of the users have haptic feedback enabled, this is enough to spend some effort on it, but not as a top priority. 

tldr: You can expect this to come, but not within the next releases. Thanks for your understanding. :-)

**Comment by MrStahlfelge** [comment]
Waiting for libgdx/libgdx#6274
